### PR TITLE
Migrate to webpack 5

### PR DESCRIPTION
**Pull request by Hyperkid123**
With this webpack config, the app is compatible with chrome 2, but should still work in the current chrome.

@john-dupuy can you trigger your test after this is deployed to ci-beta? Or can you run them against this PR?

### CODE REVIEW

**Comment by john-dupuy** [comment]
Sure, I can test locally against this PR this morning.

File: config/dev.webpack.config.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
 const { resolve } = require('path');
 const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
 const config = require('@redhat-cloud-services/frontend-components-config');
 const { config: webpackConfig, plugins } = config({
   rootFolder: resolve(__dirname, '../'),
   debug: true,
 });
+const federationPlugin = require('./federation-plugin');
+
+plugins.push(federationPlugin);
```
Comment (karelhala) [inline config/dev.webpack.config.js:11]: I'd merge this into one line
```suggestion
plugins.push(require('./federation-plugin'));
```

File: config/prod.webpack.config.js [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
 const { resolve } = require('path');
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
 const config = require('@redhat-cloud-services/frontend-components-config');
 const { config: webpackConfig, plugins } = config({
   rootFolder: resolve(__dirname, '../'),
-  debug: true,
+  mode: 'production',
 });
+const federationPlugin = require('./federation-plugin');
+plugins.push(federationPlugin);
```
Comment (karelhala) [inline config/prod.webpack.config.js:9]: Same as above

```suggestion
plugins.push(require('./federation-plugin'));
```

**Review by karelhala — APPROVED** [review]

**Review by karelhala — APPROVED** [review]
Tested locally and it works as expected!

**Pull request merged**